wazo_calld/plugins/dial_mobile/bus_consume.py

```diff
@@ -30,6 +30,7 @@ def _on_user_event(self, event):
 
         user_uuid = event['WAZO_DST_UUID']
         video_enabled = event['WAZO_VIDEO_ENABLED'] == '1'
+        ring_timeout = event.get('WAZO_RING_TIME', 30)
```

Comment:
Since the userevent PushMobile is triggered by us and have always the WAZO_RING_TIME, what do you think to use:
```suggestion
        ring_timeout = event['WAZO_RING_TIME']
```
It will remove confusion about "where the default value is set"